ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Events.cs

```diff
@@ -64,7 +64,9 @@ public void RegisterAppEvents()
     RevitApp.Application.DocumentCreated += Application_DocumentCreated;
     RevitApp.Application.DocumentCreating += Application_DocumentCreating;
     RevitApp.Application.DocumentOpened += Application_DocumentOpened;
-    RevitApp.Application.DocumentOpening += Application_DocumentOpening;
+    // This code may fire during the DocumentOpening event, in which case an exception will be
+    // thrown about being unable to subscribe to an event during that event
+    //RevitApp.Application.DocumentOpening += Application_DocumentOpening;
```

Comment:
Should we comment this out or maybe just rephrase this in the comment to say Removed hooking to DocumentOpening event because....

Reply:
Just makes it clear as opposed to commented out code, which reads ambiguous, like we might need a solution to this